Support/Scripts/run-lldb-tests.sh

```diff
@@ -187,7 +187,7 @@ if [ -n "${TARGET-}" ]; then
   if [[ "${TARGET}" == "Linux-X86_64" || "${TARGET}" == "Linux-X86_64-Clang" ]]; then
     args+=("--arch=x86_64")
     if [[ "${PLATFORM-}" = "1" ]]; then
-      args+=("--excluded" "$blacklist_dir/ds2/llvm_60.blacklist")
+      args+=("--excluded" "$blacklist_dir/ds2/llvm.blacklist")
```

Comment:
I don't think we should remove the suffix. These are related to new versions of llvm and we should still mark them separately to make resolution easier in the future.

Reply:
Meh, do you actually think it's worth maintaining which version of llvm broke a test? e.g. making a 70 and 80 file. Would value would that provide?